### PR TITLE
Embroider again again

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [ ] Check that all links work

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
deploy is broken? :shrug:

**Comment by jenweber** [comment]
This is a guess - maybe try adding a file for yourself in the `author` directory, for example see `author/jen-weber.md`. I see an error in CI about the `2022` tag but I wonder if that's a side effect. Nothing else in this PR jumps out.

**Comment by jaredgalanis** [comment]
> deploy is broken? 🤷

@NullVoxPopuli, there was some wonkiness in the original branch for the 201 Issue (it was based on a really old state of main), that you based your branch on. I'm pretty sure the deploy issues were related. Because of the wonkiness, I created a new branch and PR. Can you please try to either rebase your branch on `blog/embertimes-201-v2` or cut a new branch that's based on that and cherry-pick your commit over (that may be the easiest route, not sure).

Apologies for the further hassle! We'll get this reviewed and in the next issue. 😊

**Comment by jaredgalanis** [comment]
> > deploy is broken? 🤷
> 
> @NullVoxPopuli, there was some wonkiness in the original branch for the 201 Issue (it was based on a really old state of main), that you based your branch on. I'm pretty sure the deploy issues were related. Because of the wonkiness, I created a new branch and PR. Can you please try to either rebase your branch on `blog/embertimes-201-v2` or cut a new branch that's based on that and cherry-pick your commit over (that may be the easiest route, not sure).
> 
> Apologies for the further hassle! We'll get this reviewed and in the next issue. 😊

Update: I actually just reset the state of the original `blog/embertimes-201` branch to be update-to-date with `main` and force pushed it. You have conflicts now, but a rebase on `blog/embertimes-201` should be pretty simple.

Please ignore more my previous message about the new v2 branch.

File: content/the-ember-times-issue-201.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+And speaking of the README, it has a section of if _you_ should use embroider or not, today.
+
+> Several large, heavily-tested Ember apps are shipping to production with Embroider. So if you are excited to adopt Embroider, it is a reasonable choice. The main risks to be aware of if you choose to use Embroider in production are:
+>
+> - you're likely to discover some Ember addons don't work or break your build
+> - Embroider's own configuration options are subject to change, so you'll need to read the CHANGELOG.md when updating the Embroider packages.
+>
+> Alternatively, it is totally safe to stick with the traditional build pipeline and wait for the official cutover point when EmberCLI starts generating new apps with Embroider by default.
+Note that while some addons may break your build, this may be for the greater good. For example, instead of relying on ember-specific solutions to certain build-time behaviors, you may now follow heavily documented, and asked about, and iterated on Webpack plugins. Additionally, removing addons with build-time behavior will speed up your overall dev-server start time.
```
Comment (jaredgalanis) [inline content/the-ember-times-issue-201.md:49]: It looks like this paragraph is inheriting the block-quote from above, so it reads as if it's part of the quote from the `README.md`. I think maybe inserting a line between will fix it.

**Review by jaredgalanis — CHANGES_REQUESTED** [review]
Looks good to me. One small thing.

**Comment by angelayanpan** [comment]
just saw this. Thanks for the PR! and it's already published.  so I don't know if we can still fix anything? it should be `released mid January 2022`  not 2021

**Comment by jaredgalanis** [comment]
Thanks @angelayanpan. We can fix it on the blog site, but not in the email that was sent.

**Comment by angelayanpan** [comment]
ok, that works @jaredgalanis  thank you! are you able to update the blog? or that requires a separate PR?

**Comment by jaredgalanis** [comment]
> ok, that works @jaredgalanis  thank you! are you able to update the blog? or that requires a separate PR? 

For sure, thank you for finding it!

I'll fix it in a little bit here. 

Let us know if you are interested in writing for the Times ever. We always can use help. 😊

**Comment by jaredgalanis** [comment]
> ok, that works @jaredgalanis thank you! are you able to update the blog? or that requires a separate PR?

@angelayanpan it's fixed on the blog site now.

**Pull request merged**